### PR TITLE
NSF reporting

### PR DESCRIPTION
**Pull request by daisieh**
Oops, forgot this last piece: we would like to report journal.

### CODE REVIEW

File: fix_authors_in_json.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# fix errors in database
```
Comment (ryscher) [inline fix_authors_in_json.py]: What kind of errors? What is changed?

My python knowledge is limited, but it looks like this is printing items that need to be fixed manually, not changing them?

**Comment by ryscher** [comment]
I'm losing track of all the scripts in this directory. Can we add them all to the README, or sort them into more specific directories?

File: fix_authors_in_json.py (hunk 1 above)
Comment (daisieh) [inline fix_authors_in_json.py]: Sorry, this script wasn't supposed to be in this branch. I'll take it out.

**Comment by daisieh** [comment]
This one is sort of archival now, since the NSF pilot is over, but I wouldn't want to lose it entirely.

**Pull request merged**